docs/remote-docs.sh

```diff
@@ -88,21 +88,30 @@ function pub_pages() {
     done
 }
 
+## sed syntax is different on darwin and linux
+function sed_os(){
+    if [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
When possible, I like to test for functionality rather than platform. Is there a way to, say, run `sed --help` and differentiate the Mac version from regular sed?

Reply:
Sure, `sed --help` doesn't work on a mac so i can base it off of the exit code for `sed --help` ?

Reply:
That's really alarming! But yes, I think it would be safe to correlate "sed --help fails" with "this is not GNU sed, therefore it needs a special invocation of -i".